Sources/FileSystem/Readers.swift

```diff
@@ -67,6 +69,14 @@ public enum Readers {
 
             return result
         }
+
+        public func readTXT(from filePath: String) -> T {
```

Comment:
What values of `T` are you expecting? Will it ever be something different to `String`

Reply:
I was trying to do something like `public func readTXT<T: String>(from filePath: String) -> T {` but string can't confirm to `Decodable` so I removed it.
Do you have some thoughts on this?

Reply:
There's always overloads. If we only have need for strings atm I'd define this one as: `public func readTXT(from filePath: String) -> String` and we can add a generic one later down the track if we need it

Reply:
> There's always overloads. If we only have need for strings atm I'd define this one as: `public func readTXT(from filePath: String) -> String` and we can add a generic one later down the track if we need it

As `FileReader<T: Decodable>` has generic T:Decodable type so this odd syntax is to conform to it. Otherwise, it will give me `Cannot convert return expression of type 'String' to return type 'T'` 🤔 

If this really looks bad how about we make it as `static`?

Reply:
If you are taking @orj 's suggestion to make use of `String(contentsOf:encoding:)` we no longer need this function